Whisky/Views/Bottle Views/BottleView.swift

```diff
@@ -34,23 +34,38 @@ struct BottleView: View {
     // This just provides a way to trigger a refresh
     @State var loadStartMenu: Bool = false
     @State var showWinetricksSheet: Bool = false
+    @State private var isLoadingInstalledPrograms: Bool = false
 
     private let gridLayout = [GridItem(.adaptive(minimum: 100, maximum: .infinity))]
 
     var body: some View {
         NavigationStack(path: $path) {
             ScrollView {
-                if pins.count > 0 {
-                    LazyVGrid(columns: gridLayout, alignment: .center) {
-                        ForEach(pins, id: \.url) { pin in
-                            PinnedProgramView(bottle: bottle,
-                                              pin: pin,
-                                              loadStartMenu: $loadStartMenu,
-                                              path: $path)
+                ZStack {
```

Comment:
I don't think this ZStack is needed. Why block the user from accessing loaded pins until they've all loaded?

Reply:
Hmmm, I see what you mean - yeah, maybe a less fancy but less obstructive way of displaying the indicator would be better indeed. I'll change that, thanks for the feedback :)